etna/core/blocks/cta.py

```diff
@@ -3,27 +3,6 @@
 from wagtail import blocks
 
 
-class TimePeriodBlock(blocks.StructBlock):
-    heading = blocks.CharBlock(max_length=100, default="Explore by time period")
-
-    page = blocks.PageChooserBlock(page_type="collections.TimePeriodExplorerIndexPage")
-
-    class Meta:
-        template = "collections/blocks/time_period_explorer.html"
-        help_text = "Outputs all time period child pages"
-        icon = "th-large"
-
-
-class TopicExplorerBlock(blocks.StructBlock):
-    heading = blocks.CharBlock(max_length=100, default="Explore by topic")
-    page = blocks.PageChooserBlock(page_type="collections.TopicExplorerIndexPage")
-
-    class Meta:
-        template = "collections/blocks/topic_explorer.html"
```

Comment:
I hate to sound like a broken record, but are these templates needed any longer?

Reply:
Ah, I see they have been removed in another PR.